tasks/migration-newip-to-ts.md

```diff
@@ -110,7 +110,8 @@ The result of completing this step is:
 ### 3. Design and Application Functionality (50 points)
 
 - **(10 points)** Adaptive layout is implemented
-- **(20 points)** Your own design or improvements to the existing design are added (details should be described in the Pull Request)
+- **(10 points)** Your own design or improvements to the existing one are added (details should be described in the Pull Request)
+- **(10 points)** Additional functionality is added to the app (the PR should describe what it is, e.g. alphabetical filter or come up with your own idea)
 - **(20 points)** No remarks from the mentor regarding the design
```

Comment:
We discussed with the mentors and decided to completely remove these items; the task should be only about migrating to TypeScript.

Reply:
ty for the answer ! ^^ got it